src/dhtproto/client/mixins/NeoSupport.d

```diff
@@ -455,6 +457,40 @@ template NeoSupport ( )
             return id;
         }
 
+        /***********************************************************************
+
+            Assigns a RemoveChannel request to delete the specified channel from
+            the DHT.
+
+            See $(LINK2 dhtproto/client/request/RemoveChannel.html, dhtproto.client.request.RemoveChannel)
+            for detailed documentation.
+
+            Params:
+                channel = name of channel to remove
+                notifier = notifier delegate
+
+            Returns:
+                id of newly assigned request
+
+            Throws:
+                NoMoreRequests if the pool of active requests is full
+
+        ***********************************************************************/
+
+        public RequestId removeChannel ( cstring channel,
+            RemoveChannel.Notifier notifier )
+        {
+            auto params = Const!(Internals.RemoveChannel.UserSpecifiedParams)(
+                Const!(RemoveChannel.Args)(channel),
+                Const!(Internals.RemoveChannel.UserSpecifiedParams.SerializedNotifier)(
+                    *(cast(Const!(ubyte[notifier.sizeof])*)&notifier)
+                )
+            );
+
+            auto id = this.assign!(Internals.RemoveChannel)(params);
+            return id;
+        }
```

Comment:
Not sure if it's just the github, but while you're there, it would be nice if this commit would be before adapting the mirror tests to use this method.

Reply:
Yeah, it is.